Dev om

omiossec opened the pull request:
<!--- Provide a general summary of your changes in the Title above -->

## Description
Add a new function Get-WebServerInfo to retrieve server information from HTTP Respons header

## Related Issue
Part of #1

## Motivation and Context
Sometimes you need to retrieve server information from a website (for example a website) it's easier if we can do it directly while discussing in Slack or Team



## Types of changes
<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
- [x ] My code follows the code style of this project.
- [ x] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x ] I have read the **CONTRIBUTING** document.
- [ x] I have added tests to cover my changes.
- [x ] All new and existing tests passed.

omiossec committed:
Merge pull request #1 from poshbotio/master

Update

omiossec committed:
Add function  Get-WebServerInfo

This function retreive the server information from HTTP Response Header

devblackops commented on PoshBot.Networking/PoshBot.Networking.psm1:
```diff
@@ -106,5 +106,72 @@ function Invoke-TestPort {
+        !webserverinfo http://uri
+    #>
+    [PoshBot.BotCommand(
+        CommandName = 'testport',
```
Copy paste error? This should be something like `webserverinfo`.

devblackops commented on PoshBot.Networking/PoshBot.Networking.psm1:
```diff
@@ -106,5 +106,72 @@ function Invoke-TestPort {
+    finally {
+
+        if ($ErrorMessage) {
+            New-PoshBotCardResponse -Type Warning -Text "$ErrorMessage :(" -Title 'Rut row' -ThumbnailUrl 'http://images4.fanpop.com/image/photos/17000000/Scooby-Doo-Where-Are-You-The-Original-Intro-scooby-doo-17020515-500-375.jpg'
```
I have the thumbnails used by the builtin plugin hosted in the main PoshBot repo. You can use this link so we're not relying on the other site.

`https://raw.githubusercontent.com/poshbotio/PoshBot/master/Media/scooby_doo.jpg`

omiossec replied:
Ok I will change the 2 points

omiossec committed:
Add get-geolocip  get geo data from Ripe Database to locate an IP or a network in the World

omiossec committed:
Merge pull request #2 from omiossec/master

Add get-geolocip  get geo data from Ripe Database to locate an IP or …

omiossec committed:
Add get-NetASInfo

get-NetASInfo is function to retreive AS information from Ripe, AS Number and Company information

omiossec committed:
Update error and image

Change command name webserverinfo
Change Image URI

devblackops commented:
Thanks @omiossec! Looks like the other PRs were duplicated in this one I'm going to close those out.
